packages/extension-chakra-storefront/src/hooks/use-config.js

```diff
@@ -23,7 +23,8 @@ export const useConfig = () => {
     const extension = useMemo(
         () =>
             applicationExtensions.find(
-                (extension) => extension.constructor.id === `@salesforce/extension-chakra-storefront`
+                (extension) =>
+                    extension.constructor.id === `@salesforce/extension-chakra-storefront`
```

Comment:
Fixed some linting errors in the chakra-storefront. As I mentioned, there are failing unit tests for the storefront, and I'll ignore them for this PR.